internal/pkg/api/server/buffer/service/service.go

```diff
@@ -41,6 +42,19 @@ func (*service) HealthCheck(dependencies.ForPublicRequest) (res string, err erro
 func (*service) CreateReceiver(d dependencies.ForProjectRequest, payload *buffer.CreateReceiverPayload) (res *buffer.Receiver, err error) {
 	ctx, store := d.RequestCtx(), d.ConfigStore()
 
```

Comment:
Tu mozes pridat zamok, ak chces zabezpecit presne maximum `100`, pouzival by sa pri create a delete.
Kedze je to operacia, ktora sa nevola casto (niekolko krat za sekundu), tak je to v pohode.

Mne to nevadi ani bez zamku a moze sa stat, ze sa vytvori `102+` receivers, ale urcite nie `10000`.
Ale ak ti to pride problem, tak zamok je jednoduche riesenie.

Reply:
Mně to jako problém nepřijde, že bude ~102 místo 100

Reply:
ok.